### PR TITLE
[Notification Triggers article] Revive Notification Triggers

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- There will be a second OT.

### CODE REVIEW

**Comment by tomayac** [comment]
@rknoll Please let me know once the new OT link is live. I assume we don't recycle the [old one](https://developers.chrome.com/origintrials/#/view_trial/6883752030435803137), do we?

File: src/site/content/en/blog/notification-triggers/index.md [hunk 1]
```diff
@@ -71,27 +71,36 @@ alarms for telephone conferences or video calls.
 | -------------------------------------------- | ---------------------------- |
 | 1. Create explainer                          | [Complete][explainer]        |
 | 2. Create initial draft of specification     | Not started                  |
-| **3. Gather feedback and iterate on design** | **[In progress](#feedback)** |
-| 4. Origin trial                              | [Complete][ot]               |
+| 3. Gather feedback and iterate on design.    | [In progress](#feedback)     |
+| **4. Origin trial**                          | [In progress][ot]            |
```
Comment (tomayac) [inline src/site/content/en/blog/notification-triggers/index.md:75]:
```suggestion
| **4. Origin trial**                          | **[In progress][ot]**        |
```

File: src/site/content/en/blog/notification-triggers/index.md [hunk 2]
```diff
@@ -71,27 +71,36 @@ alarms for telephone conferences or video calls.
...
+
+Starting in Chrome&nbsp;86, the Notification Triggers API will be available
+as an origin trial in Chrome. The origin trial is expected to end in
+Chrome&nbsp;88 (February&nbsp;24, 2021).
+
+{% Aside %}
+  An earlier origin trial for the feature, which gave developers a first chance
+  to try out the proposed API, ran from Chrome&nbsp;80 to&nbsp;83
+  (see the summary of the
+  [obtained feedback](https://docs.google.com/document/d/1Nl1emEqxjTzPLNIAPiS26Vtq3mBdNyCxfMY6QwaD45s/edit)).
```
Comment (tomayac) [inline src/site/content/en/blog/notification-triggers/index.md:95]:
```suggestion
  to try out the proposed API, ran from Chrome&nbsp;80 to&nbsp;83.
  You can read the summary of the
  [obtained feedback](https://docs.google.com/document/d/1Nl1emEqxjTzPLNIAPiS26Vtq3mBdNyCxfMY6QwaD45s/edit))
  so far.
```

File: src/site/content/en/blog/notification-triggers/index.md [hunk 3]
```diff
@@ -71,27 +71,37 @@ alarms for telephone conferences or video calls.
...
+
+Starting in Chrome&nbsp;86, the Notification Triggers API will be available
+as an origin trial in Chrome. The origin trial is expected to end in
+Chrome&nbsp;88 (February&nbsp;24, 2021).
+
+{% Aside %}
+  An earlier origin trial for the feature, which gave developers a first chance
+  to try out the proposed API, ran from Chrome&nbsp;80 to&nbsp;83.
+  You can read the summary of the
+  [obtained feedback](https://docs.google.com/document/d/1Nl1emEqxjTzPLNIAPiS26Vtq3mBdNyCxfMY6QwaD45s/edit))
```
Comment (jpmedley) [inline src/site/content/en/blog/notification-triggers/index.md:95]:
```suggestion
  [feedback obtained](https://docs.google.com/document/d/1Nl1emEqxjTzPLNIAPiS26Vtq3mBdNyCxfMY6QwaD45s/edit))
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/notification-triggers/index.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ description:
   The Notification Triggers API allows developers to schedule local notifications that don't require
   a network connection, which makes them ideal for use cases like calendar apps.
 date: 2019-10-24
-updated: 2020-08-23
+updated: 2020-09-24
```
Comment (tomayac) [inline src/site/content/en/blog/notification-triggers/index.md:12]:
```suggestion
updated: 2020-09-25
```

**Comment by tomayac** [comment]
Please **do not merge** this yet, we're still waiting for the OT to show up. Thanks.

**Review by jpmedley — APPROVED** [review]
Waiting on OT. Please do not merge.

**Comment by jpmedley** [comment]
@tomayac, are you just looking for someone to squash and merge?

**Comment by tomayac** [comment]
> @tomayac, are you just looking for someone to squash and merge?

Waiting for Notification Triggers to re-appear on https://developers.chrome.com/origintrials/#/trials/active.

**Comment by rknoll** [comment]
> Waiting for Notification Triggers to re-appear on https://developers.chrome.com/origintrials/#/trials/active.

Just appeared as active again 😃
https://developers.chrome.com/origintrials/#/view_trial/6883752030435803137

**Comment by tomayac** [comment]
> > Waiting for Notification Triggers to re-appear on https://developers.chrome.com/origintrials/#/trials/active.
> 
> Just appeared as active again 😃
> https://developers.chrome.com/origintrials/#/view_trial/6883752030435803137

Thanks, @rknoll. Merged now.

**Pull request merged**